site/reference/pkg/cat/README.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Cat"
+linkTitle: "cat"
+type: docs
+description: >
+  Print the KRM resources in a file/directory
+---
+
+<!--mdtogo:Short
+    Print the KRM resources in a file/directory
+-->
+
+Cat prints the resources in a package as yaml to stdout.
+
+Cat is useful for printing the KRM resources in a file/directory which might
+contain other non-resource files.
+
+### Synopsis
+
+<!--mdtogo:Long-->
+
+```
+kpt pkg cat [FILE | DIR]
+
+DIR:
+  Path to a directory with KRM resource files. Defaults to the current working directory.
+
+FILE:
+  Path to a KRM file.
```

Comment:
I don't think we use "KRM file" in other places. KRM is obvious to us but necessarily to the end user. As you're auditing the CLI usability and reference docs, make sure we use consistent nomenclature. Related to "config vs resource" discussions.

Reply:
renamed it to `resources`. Will make it consistent all across the docs.